### ISSUE TITLE
Add feature to add actor to conversation with name hidden

### ISSUE THREAD

**Issue by Celldweller** [CristianVasile23/conversation-hud#71]
There is currently no easy way to add an actor to a conversation on-the-fly without its name being displayed. It can only be done retrospectively.

If you add an actor via drag and drop, you should be able to add it “without a name” using a shortkey.
For example, by dragging and dropping while holding down the SHIFT key (like the CTRL key to replace). 

This would make it much easier to create spontaneous dialogs during the session in which new NPCs appear that the players do not yet know.

Alternatively, there could be a toggle button “Add with hidden name”.

**Comment by CristianVasile23** [comment]
You are right, that is indeed a big omission in terms of functionality. I will add something like this in the future, but that will most likely happen once the rework is out as that is my focus at the moment.
